modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -0,0 +1,98 @@
+# Frequency Domain Dynamics
+
+The following example presents a frequency domain analysis done in the MOOSE Tensor Mechanics module. A frequency domain analysis provides the structural response at a discrete frequencies where for each frequency, an independent steady state simulation is performed. This document provides an example of modeling a dynamic problem at a single frequency (time-harmonic problem).
+
+Frequency domain analysis is often used to determine a frequency response function (FRF). An FRF describes the relationship between an input (frequency of the input forcing source) and output (displacement response of a system). For simple systems, an analytic FRF can be derived. For more complex systems, FRF is numerically obtained by sweeping over frequency range of interest and plotting frequency vs. the response of the system. The frequencies correspond to the peaks on the FRF plot are the natural modes of the system (the eigen/fundamental frequncies), whereas the displacement profiles correspond to those eigen freqeuncies are the mode shapes (eigen vectors).
+
+Other applications of frequency domain dynamics are: (1) computation of band structure (dispersion curves) of lattices/metamaterial, (2) inverse design for vibration control, e.g. design a system so that it has as minimum/maximum response at particular frequencie/s, (3) material properties inversion/optimization given discrete responses.
+
+On the other hand, frequency domain analyses can be advantageous over time domain counterpart in several cases, for example, when the frequency spectrum of a signal consists of a few frequencies, or, when it is needed to have a better control of the numerical dispersion.
```

Comment:
Frequency domain analyses can be advantageous over its time domain counterpart in several cases.  For example, when signals frequency spectrum consists of a few frequencies, this requires only a few steady state simulations at each frequency of interest as opposed to an entire transient simulation over the scale of interest.  Frequency domain analysis is also able to better control numerical dispersion.